script.skipit/resources/skins/default/1080i/skipitwindow.xml

```diff
@@ -0,0 +1,32 @@
+<window id="1191" type="dialog">
```

Comment:
Please name this script-skipitwindow.xml

Reply:
To avoid clashing with skin files

Reply:
Done